src/backend/distributed/worker/worker_drop_protocol.c

```diff
@@ -177,7 +177,7 @@ WorkerDropDistributedTable(Oid relationId)
 	 * until the user runs DROP EXTENSION. Therefore, we skip dropping the
 	 * table.
 	 */
-	if (!IsObjectAddressOwnedByExtension(&distributedTableObject, NULL))
+	if (!IsAnyObjectAddressOwnedByExtension(list_make1(&distributedTableObject), NULL))
```

Comment:
would be good to avoid passing reference to a local variable again, maybe convert distributedTableObject to a pointer